### DISCUSSION TITLE
History filter "by app"

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#355] [Q&A]
Hi,

This weekend I tested an app and ended up this morning with a big mess in my history 😩. 
All shows I once watched have been duplicated and re-added. Now it seems like I watched them all in a 2 days span.

I'm going to clean up the mess with a script or something but I would find it useful to be able to filter the history "by app" that made the action (checkin or watched). This way, users could pinpoint the apps that create those kind of issues and remove/clean their history faster.

Cheers,

Kevin

**Comment by rectifyer** [comment]
We don't include this info for privacy reasons, but maybe we can allow it for only the OAuth user. I'll discuss internally and see if this is something we want to do.
